specification/metrics/listunitprice.md

```diff
@@ -14,7 +14,7 @@ List Unit Price
 
 ## Description
 
-Represents the suggested provider published price for a single pricing measurement unit, which incorporates volume/tier-based reductions while excluding any negotiated or commitment-based discounts.
+A suggested provider-published unit price for a single pricing measurement unit (identified by a [Sku Price Id](#skupriceid)), which incorporates free-tier, volume/tier-based, BYOL-based and dynamically priced SKU rates, while excluding any other discounts or the amortization of upfront charges (one-time or recurring).
```

Comment:
```suggestion
A suggested provider-published unit price, identified by the [SKU Price ID](#skupriceid), for a single [Pricing Unit](#pricingunit) of the associated SKU of a charge.
```

Reply:
I would also remove "A" in the beginning. It's not needed.